configure.py

```diff
@@ -445,7 +445,7 @@ def process_command_line(args): # pylint: disable=too-many-locals
     install_group.add_option('--prefix', metavar='DIR',
                              help='set the install prefix')
     install_group.add_option('--destdir', metavar='DIR',
-                             help='set the install directory')
+                             help='set the install directory (deprecated, use DESTDIR environment variable)')
```

Comment:
Not "deprecated" but "removed" because it does not work anymore. Btw. the help text for "--via-amalgamation" is outdated now, so this is not a good orentation.